alf/algorithms/distributed_off_policy_algorithm.py

```diff
@@ -528,8 +535,9 @@ def _train_iter_off_policy(self):
             logging.debug(f"Rank {self._ddp_rank} sends params to unrollers "
                           f"{self._unrollers_to_update_params}")
             for unroller_id in self._unrollers_to_update_params:
-                if not self._send_params_to_unroller(unroller_id):
-                    dead_unrollers.append(unroller_id)
+                with record_time("time/trainer_send_params_to_unroller"):
```

Comment:
should be outside the "if" at line 532 so that it can correctly compute the average time spent for each iteration.